source/WB-CAN-Firmware.c

```diff
@@ -196,10 +293,34 @@ static void testTask(void *pv) {
 	}
 }
 
+
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
+
+
+
+/*******************************************************************************
+ * Tasks
+ ******************************************************************************/
+static void blinkTask(void *pv) {
+	while(1) {
+
+		vTaskDelay(pdMS_TO_TICKS(500));
+//		printf("BLINK");
+	}
+}
+
+
 static void actuatorTask(void *pv){
 	uint32_t period = 100;
 	for(;;){
-		PWM(period, duty_cycle); // Uses Global Variable changed in Control Task
+		if (uxSemaphoreGetCount( semaphore_PWMActive ) == 1) {
```

Comment:
Is this a counting semaphore? If it's just binary, the `== 1` is redundant. If it *is*, I think it should probably be a binary semphore.

```suggestion
		if (uxSemaphoreGetCount(semaphore_PWMActive )) {
```